nextdns/resource_nextdns_profile.go

```diff
@@ -43,6 +44,26 @@ func resourceNextDNSProfileCreate(ctx context.Context, d *schema.ResourceData, m
 	return resourceNextDNSProfileRead(ctx, d, meta)
 }
 
+func formatProfileIPv6Suffix(profileID []byte) []byte {
```

Comment:
I don't think we need this since this information is returned by the `/setup` API today. I will add this to the Go client later today or tomorrow (_today it doesn't cover it_) so we can consume what the API gives us to use. If at some point this changes in the future, no changes are required on the `terraform` provider.